Week2/homework/js-exercises/1-correct-the-sentence.js

```diff
@@ -0,0 +1,8 @@
+"use strict";
+let myString = 'hello,this,is,a,difficult,to,read,sentence';
+
+console.log(myString.length);
+
+let myNewString = myString.replace(/,/g, ' ');
+
+console.log(myNewString);  
```

Comment:
:+1:

You can also acheive the same without using regular expressions by using `.split()` and `.join()` methods on `String`. You can look up MDN on how they work :)